src/react/ui/pages/ClusterView/ResourceChart.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { AreaChart, Area, XAxis, YAxis, CartesianGrid, Tooltip,  ResponsiveContainer } from 'recharts';
+import moment from "moment";
+import { ClusterViewNode } from "../../../engine/helpers/nodesStateUpdate";
+
+const timeFormatter = (item: number) => moment(item).format("mm:ss");
+
+// const CustomTooltip = ({ active, payload }) => {
+//     if (active && payload && payload.length > 0 ) {
+//         return (
+//            <div className="customTooltip">
+//                 <div style={{ ...tooltipStyle }}>
+//                     Read Time: `{moment(payload[0].payload.timestamp).format("hh:mm:ss")}`
+//                 </div>
+//                 <div style={{ ...tooltipStyle, color: payload[3].stroke }}>Available: {`${payload[3].value}${payload[3].unit}`}</div>
+//                 <div style={{ ...tooltipStyle, color: payload[2].stroke }}>In use: {`${payload[2].value}${payload[2].unit}`}</div>
+//                 <div style={{ ...tooltipStyle, color: payload[1].stroke }}>Allocated: {`${payload[1].value}${payload[1].unit}`}</div>
+//            </div>
+//
+//         );
+//     }
+//     return null;
+// }
+
```

Comment:
is custom tooltip commented temporary? The default tooltip one was not friendly...